SPECS/kernel/kernel.spec

```diff
@@ -2,6 +2,12 @@
 %global sha512hmac bash %{_sourcedir}/sha512hmac-openssl.sh
 %define uname_r %{version}-%{release}
 
+# Edit default debuginfo settings
```

Comment:
Seeing how this is uncommon, could you expand the comment to say what this does and why we do it?

Reply:
+1 a bit more comment in the spec would likely be helpful in the future, otherwise LGTM